Packs/DSPM/Integrations/DSPM/DSPM.yml

```
@@ -0,0 +1,495 @@
+category: Network Security
+commonfields:
+  id: DSPM
+  version: -1
+configuration:
+- defaultvalue: https://api.dig.security/
+  display: Your DSPM server URL
+  name: dspmBaseUrl
+  required: true
+  type: 0
+- displaypassword: DSPM API Key
+  additionalinfo: API key to use for the connection
+  name: dspmApiKey
+  required: true
+  hiddenusername: true
+  type: 9
+- display: Default Slack user for notifications
+  additionalinfo: the default user to receive Slack notifications in case of any errors.
+  name: defaultSlackUser
+  required: true
+  type: 0
+- display: Lifetime for slack notification ( in hours)
+  name: slackMsgLifetime
+  required: true
+  defaultvalue: 2
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+description: |
+  Remediate your data security risks. Integrate with Prisma Cloud DSPM to fetch your data security risks and remediate them with OOTB playbooks.
+display: Prisma Cloud DSPM
+name: DSPM
+script:
+  commands:
+  - arguments:
+    - description: 'List of rule names'
+      isArray: true
+      isOptional: true
+      name: ruleNameIn
+      required: false
+      secret: false
+    - description: 'Exact rule name'
+      isArray: false
+      isOptional: true
+      name: ruleNameEqual
+      required: false
+      secret: false
+    - description: 'List of DSPM tag keys'
+      isArray: true
+      isOptional: true
+      name: dspmTagKeyIn
+      required: false
+      secret: false
+    - description: 'Exact DSPM tag key'
+      isArray: false
+      isOptional: true
+      name: dspmTagKeyEqual
+      required: false
+      secret: false
+    - description: 'List of DSPM tag values'
+      isArray: true
+      isOptional: true
+      name: dspmTagValueIn
+      required: false
+      secret: false
+    - description: 'Exact DSPM tag value'
+      isArray: false
+      isOptional: true
+      name: dspmTagValueEqual
+      required: false
+      secret: false
+    - description: 'List of project IDs'
+      isArray: true
+      isOptional: true
+      name: projectIdIn
+      required: false
+      secret: false
+    - description: 'Exact project ID'
+      isArray: false
+      isOptional: true
+      name: projectIdEqual
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]'
+      isArray: false
+      isOptional: true
+      name: cloudProviderIn
+      required: false
+      secret: false
```

Try to use single-select or multi-select in parameters (or single-select with command arguments) when predifined values are pre-known rather than free form strings
do it for all the arguments that needed

```suggestion
    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]'
      isArray: false
      isOptional: true
      name: cloudProviderIn
      required: false
      secret: false
      auto: PREDEFINED
      predefined:
      - 'AWS'
      - 'AZURE'
      defaultValue: 'AWS'
```